Remove broken and inactive links, fix redirected links, and add a link to the community calendar (#1240)
ipfs/ipfs-docs: docs/concepts/usage-ideas-examples.md

### Desktop applications

> [IPFS Desktop](https://github.com/ipfs/ipfs-desktop) is the official desktop client for IPFS. It comes with a built-in IPFS node that lets you pin files and gives you a link to share them. This is generally seen as the easiest point of entry into the IPFS ecosystem.

Reviewer: ```suggestion
[IPFS Desktop](https://github.com/ipfs/ipfs-desktop) is the official desktop client for IPFS. It comes with a built-in IPFS node that lets you pin files and gives you a link to share them. This is one of the easiest ways to start pinning your files to IPFS.
```
simplify

### Hosting software containers

> Netflix is on its way to using IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.io/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than traditional synchronization methods.

Reviewer 2: @DannyS03 in future edits, use .tech.

We moved blog, docs, and main website  to https://blog.ipfs.tech  :-)  and want .io to be only used for gateway at `ipfs.io/ipfs`

Author: Fixed in 06b83516c7f81c391800f65321e9ce259a3c6742

### IPFS hosting with Textile

> [Textile](https://linktr.ee/textileio) is a hosting company on IPFS developing the layers on top of IPFS. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).

Reviewer 3: ```suggestion
[Textile](https://linktr.ee/textileio) is developing the layers on top of IPFS to extend its hosting capabilities. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).
```
